cli/template/extras/src/server/api/trpc/with-auth-prisma.ts

```diff
@@ -53,7 +53,7 @@ export const createTRPCContext = async (opts: CreateNextContextOptions) => {
   // Get the session from the server using the unstable_getServerSession wrapper function
   const session = await getServerAuthSession({ req, res });
```

Comment:
Since this invocation is async, we can't actually remove the async modifier from the method.

Reply:
You're right, sorry about that. But the problem still remains since [this change](https://github.com/t3-oss/create-t3-app/pull/1060/files#diff-c4d6c739a033f78862d3c37155af6024deacbf7e77c7e8aa47217d9d30acaa78R68) removed the Awaited<ReturnType<...>> from the trpc context call, even in the auth version.

Reply:
Yea that's new from trpc 10.8.1 that you can pass either an object, or a sync/async function to it and trpc will unwrap the type automatically

https://github.com/trpc/trpc/pull/3495